media/mp4.ksy

```diff
@@ -0,0 +1,408 @@
+meta:
+  id: mp4
+  file-extension: mp4
+  endian: be
+  license: CC0-1.0
```

Comment:
At the very least, we'll need `title` here so it would be possible to find this file on the website.